lab_7_llm/main.py

```diff
@@ -1,11 +1,16 @@
 """
-Neural machine translation module.
+Neural summarization module.
```

Comment:
Checkout the files from 7 lab to the master

Reply:
done